Sharprompt/Internal/OffscreenBuffer.cs

```diff
@@ -16,7 +16,7 @@ public OffscreenBuffer(IConsoleDriver consoleDriver)
         }
 
         private readonly IConsoleDriver _consoleDriver;
-        private readonly List<List<TextInfo>> _outputBuffer = new();
+        private readonly List<List<TextInfo>> _outputBuffer = new List<List<TextInfo>>();
```

Comment:
This change is not necessary.

Reply:
Upps I think this change was made automatically by Visual Studio.